mods/d2k/rules/player.yaml

```diff
@@ -87,7 +87,6 @@ Player:
 		Prerequisites: techlevel.low, techlevel.medium, techlevel.high, techlevel.superweapons
 		Id: unrestricted
 	EnemyWatcher:
-	HarvesterInsurance:
```

Comment:
Why i did this is because original game didn't have such thing in Skirnish/MP, it is a Mission Logic and what HarvesterInsurance currently does is not same as Original. It needs to be rewrote.